dist/common/scripts/scylla_io_setup

```diff
@@ -177,7 +177,8 @@ def run_iotune():
                 subprocess.check_call([bindir() + "/iotune",
                                        "--format", "envfile",
                                        "--options-file", etcdir() + "/scylla.d/io.conf",
-                                       "--properties-file", etcdir() + "/scylla.d/io_properties.yaml"] + iotune_args)
+                                       "--properties-file", etcdir() + "/scylla.d/io_properties.yaml",
+                                       *(f"--get-best-iops-with-buffer-sizes={s}" for s in [512, 4096])] + iotune_args)
```

Comment:
Actually, the `--get-best-iops-with-buffer-sizes` is a tool for developers to research how disk behaves.

Here we have two BUGs.

First -- iotune doesn't use the physical_io_size at all, but it most likely should (for write IOPS test)  (https://github.com/scylladb/seastar/issues/3006)
Second -- there's known confirmed bug that AWS instances lie about their physical sizes. For that, scylla_io_setup must explicitly instruct iotune to use some specific value for write IOPS measurement, not ask it to do fancy guessing techniques (or maybe it's even iotune that should detect that it runs on AWS disk and update itself with a hard-coded quirk, but I'm not sure it's very good idea).

Reply:
Yes, it creates buffers sized and aligned to the same value